packages/binding-http/src/credential.ts

```diff
@@ -167,9 +167,9 @@ export class TuyaCustomBearer extends Credential {
     protected key: string;
     protected secret: string;
     protected baseUri: string;
-    protected token: string;
-    protected refreshToken: string;
-    protected expireTime: Date;
+    protected token: string | undefined;
```

Comment:
I am just wondering.. would the following not work?

`protected token?: string;`

OR do you want to convey that `token` is **not** optional?

Reply:
addressed with https://github.com/eclipse-thingweb/node-wot/pull/1048/commits/5469a3ada4a6da4a368388bc02a4aa94276caf38 (I checked the whole file to have consistency, but in other places we might use the old way)